provisioning/service/src/Config/IndividualEnrollment.cs

```diff
@@ -153,12 +154,19 @@ public IndividualEnrollment(string registrationId, Attestation attestation)
             DateTime lastUpdatedDateTimeUtc,
             string eTag)
         {
+            /* SRS_INDIVIDUAL_ENROLLMENT_21_003: [The constructor shall throws ProvisioningServiceClientException if one of the 
+                                                    provided parameters in JSON is not valid.] */
             if (string.IsNullOrWhiteSpace(eTag))
             {
                 throw new ProvisioningServiceClientException("Service respond an individualEnrollment without eTag.");
             }
+            if (attestation == null)
```

Comment:
Nit: new line after closing brace (except when not followed by another `}`)

Reply:
Here and everywhere else.

Reply:
Done